quartz.layout.ts

```diff
@@ -16,7 +16,7 @@ export const sharedPageComponents: SharedLayout = {
 
 // components for pages that display a single page (e.g. a single note)
 export const defaultContentPageLayout: PageLayout = {
-  beforeBody: [
+  pageHeader: [
```

Comment:
eh let's just keep it as beforeBody, to avoid too much breaking.

then we can add docs about this

Reply:
bring back beforeBody instead of pageHeader